[2017-09-12 19:23:30] console.ERROR: Error thrown while running command "doctrine:schema:update --force". Message: "An exception occured in driver: SQLSTATE[42000] [1049] Unknown database 'symfony'" {"error":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[42000] [1049] Unknown database 'symfony' at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1049): SQLSTATE[42000] [1049] Unknown database 'symfony' at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:47, PDOException(code: 1049): SQLSTATE[42000] [1049] Unknown database 'symfony' at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:43)","command":"doctrine:schema:update --force","message":"An exception occured in driver: SQLSTATE[42000] [1049] Unknown database 'symfony'"} []
[2017-09-12 19:23:30] console.DEBUG: Command "doctrine:schema:update --force" exited with code "1" {"command":"doctrine:schema:update --force","code":1} []
[2017-09-12 19:24:08] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2017-09-12 19:24:09] doctrine.DEBUG: SHOW FULL TABLES WHERE Table_type = 'BASE TABLE' [] []
[2017-09-12 19:24:09] doctrine.DEBUG: CREATE TABLE oauth2_access_tokens (id INT AUTO_INCREMENT NOT NULL, client_id INT NOT NULL, user_id INT DEFAULT NULL, token VARCHAR(255) NOT NULL, expires_at INT DEFAULT NULL, scope VARCHAR(255) DEFAULT NULL, UNIQUE INDEX UNIQ_D247A21B5F37A13B (token), INDEX IDX_D247A21B19EB6921 (client_id), INDEX IDX_D247A21BA76ED395 (user_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-09-12 19:24:09] doctrine.DEBUG: CREATE TABLE oauth2_auth_codes (id INT AUTO_INCREMENT NOT NULL, client_id INT NOT NULL, user_id INT DEFAULT NULL, token VARCHAR(255) NOT NULL, redirect_uri LONGTEXT NOT NULL, expires_at INT DEFAULT NULL, scope VARCHAR(255) DEFAULT NULL, UNIQUE INDEX UNIQ_A018A10D5F37A13B (token), INDEX IDX_A018A10D19EB6921 (client_id), INDEX IDX_A018A10DA76ED395 (user_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-09-12 19:24:10] doctrine.DEBUG: CREATE TABLE oauth2_clients (id INT AUTO_INCREMENT NOT NULL, random_id VARCHAR(255) NOT NULL, redirect_uris LONGTEXT NOT NULL COMMENT '(DC2Type:array)', secret VARCHAR(255) NOT NULL, allowed_grant_types LONGTEXT NOT NULL COMMENT '(DC2Type:array)', PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-09-12 19:24:10] doctrine.DEBUG: CREATE TABLE oauth2_refresh_tokens (id INT AUTO_INCREMENT NOT NULL, client_id INT NOT NULL, user_id INT DEFAULT NULL, token VARCHAR(255) NOT NULL, expires_at INT DEFAULT NULL, scope VARCHAR(255) DEFAULT NULL, UNIQUE INDEX UNIQ_D394478C5F37A13B (token), INDEX IDX_D394478C19EB6921 (client_id), INDEX IDX_D394478CA76ED395 (user_id), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-09-12 19:24:10] doctrine.DEBUG: CREATE TABLE users (id INT AUTO_INCREMENT NOT NULL, username VARCHAR(180) NOT NULL, username_canonical VARCHAR(180) NOT NULL, email VARCHAR(180) NOT NULL, email_canonical VARCHAR(180) NOT NULL, enabled TINYINT(1) NOT NULL, salt VARCHAR(255) DEFAULT NULL, password VARCHAR(255) NOT NULL, last_login DATETIME DEFAULT NULL, confirmation_token VARCHAR(180) DEFAULT NULL, password_requested_at DATETIME DEFAULT NULL, roles LONGTEXT NOT NULL COMMENT '(DC2Type:array)', UNIQUE INDEX UNIQ_1483A5E992FC23A8 (username_canonical), UNIQUE INDEX UNIQ_1483A5E9A0D96FBF (email_canonical), UNIQUE INDEX UNIQ_1483A5E9C05FB297 (confirmation_token), PRIMARY KEY(id)) DEFAULT CHARACTER SET utf8 COLLATE utf8_unicode_ci ENGINE = InnoDB [] []
[2017-09-12 19:24:11] doctrine.DEBUG: ALTER TABLE oauth2_access_tokens ADD CONSTRAINT FK_D247A21B19EB6921 FOREIGN KEY (client_id) REFERENCES oauth2_clients (id) [] []
[2017-09-12 19:24:11] doctrine.DEBUG: ALTER TABLE oauth2_access_tokens ADD CONSTRAINT FK_D247A21BA76ED395 FOREIGN KEY (user_id) REFERENCES users (id) [] []
[2017-09-12 19:24:11] doctrine.DEBUG: ALTER TABLE oauth2_auth_codes ADD CONSTRAINT FK_A018A10D19EB6921 FOREIGN KEY (client_id) REFERENCES oauth2_clients (id) [] []
[2017-09-12 19:24:12] doctrine.DEBUG: ALTER TABLE oauth2_auth_codes ADD CONSTRAINT FK_A018A10DA76ED395 FOREIGN KEY (user_id) REFERENCES users (id) [] []
[2017-09-12 19:24:12] doctrine.DEBUG: ALTER TABLE oauth2_refresh_tokens ADD CONSTRAINT FK_D394478C19EB6921 FOREIGN KEY (client_id) REFERENCES oauth2_clients (id) [] []
[2017-09-12 19:24:12] doctrine.DEBUG: ALTER TABLE oauth2_refresh_tokens ADD CONSTRAINT FK_D394478CA76ED395 FOREIGN KEY (user_id) REFERENCES users (id) [] []
[2017-09-12 19:25:30] doctrine.DEBUG: "START TRANSACTION" [] []
[2017-09-12 19:25:30] doctrine.DEBUG: INSERT INTO users (username, username_canonical, email, email_canonical, enabled, salt, password, last_login, confirmation_token, password_requested_at, roles) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) {"1":"admin","2":"admin","3":"admin@admin.com","4":"admin@admin.com","5":true,"6":"M.KDKuPS95.Q/bHxw4uXsNBDAU [...]","7":"8vyLQz28Gs7yUWKxqQlRmcrlVZ [...]","8":null,"9":null,"10":null,"11":[]} []
[2017-09-12 19:25:30] doctrine.DEBUG: "COMMIT" [] []
[2017-09-12 19:26:53] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:27:03] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2017-09-12 19:27:03] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:04] request.CRITICAL: Exception thrown when handling an exception (Symfony\Component\Debug\Exception\ContextErrorException: Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:04] php.CRITICAL: Uncaught Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:04] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:04] request.CRITICAL: Exception thrown when handling an exception (Symfony\Component\Debug\Exception\ContextErrorException: Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:24] request.INFO: Matched route "app_api_index". {"route":"app_api_index","route_parameters":{"_controller":"AppBundle\\Controller\\ApiController::indexAction","_route":"app_api_index"},"request_uri":"http://127.0.0.1:8000/api","method":"GET"} []
[2017-09-12 19:27:25] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:25] request.CRITICAL: Exception thrown when handling an exception (Symfony\Component\Debug\Exception\ContextErrorException: Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:25] php.CRITICAL: Uncaught Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:25] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80, Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:27:25] request.CRITICAL: Exception thrown when handling an exception (Symfony\Component\Debug\Exception\ContextErrorException: Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:28:40] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:28:43] request.INFO: Matched route "app_api_index". {"route":"app_api_index","route_parameters":{"_controller":"AppBundle\\Controller\\ApiController::indexAction","_route":"app_api_index"},"request_uri":"http://127.0.0.1:8000/api","method":"GET"} []
[2017-09-12 19:28:43] security.INFO: An AuthenticationException was thrown; redirecting to authentication entry point. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AuthenticationCredentialsNotFoundException(code: 0): A Token was not found in the TokenStorage. at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/Security/Http/Firewall/AccessListener.php:53)"} []
[2017-09-12 19:28:43] security.DEBUG: Calling Authentication entry point. [] []
[2017-09-12 19:28:47] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2017-09-12 19:28:48] security.INFO: An AuthenticationException was thrown; redirecting to authentication entry point. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AuthenticationCredentialsNotFoundException(code: 0): A Token was not found in the TokenStorage. at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/Security/Http/Firewall/AccessListener.php:53)"} []
[2017-09-12 19:28:48] security.DEBUG: Calling Authentication entry point. [] []
[2017-09-12 19:28:50] request.INFO: Matched route "homepage". {"route":"homepage","route_parameters":{"_controller":"AppBundle\\Controller\\DefaultController::indexAction","_route":"homepage"},"request_uri":"http://127.0.0.1:8000/","method":"GET"} []
[2017-09-12 19:28:50] security.INFO: An AuthenticationException was thrown; redirecting to authentication entry point. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AuthenticationCredentialsNotFoundException(code: 0): A Token was not found in the TokenStorage. at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/Security/Http/Firewall/AccessListener.php:53)"} []
[2017-09-12 19:28:50] security.DEBUG: Calling Authentication entry point. [] []
[2017-09-12 19:29:15] request.ERROR: Uncaught PHP Exception Symfony\Component\HttpKernel\Exception\NotFoundHttpException: "No route found for "GET //oauth/v2/token"" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/classes.php line 3748 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET //oauth/v2/token\" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/classes.php:3748, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0):  at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainerUrlMatcher.php:150)"} []
[2017-09-12 19:29:15] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:29:19] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://127.0.0.1:8000/oauth/v2/token","method":"GET"} []
[2017-09-12 19:30:52] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://127.0.0.1:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:30:53] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:30:53] php.CRITICAL: Fatal Error: Call to a member function loadUserByUsername() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:30:53] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function loadUserByUsername() on null" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 161 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:30:53] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:32:18] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://127.0.0.1:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:34:05] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://127.0.0.1:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:34:05] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:34:05] php.CRITICAL: Fatal Error: Call to a member function loadUserByUsername() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:34:05] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function loadUserByUsername() on null" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 161 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:34:05] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:38:02] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://127.0.0.1:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:38:04] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:38:04] php.CRITICAL: Fatal Error: Call to a member function loadUserByUsername() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:38:04] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function loadUserByUsername() on null" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 161 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:38:04] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:38:47] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://localhost:8000/app_dev.php/oauth/v2/token","method":"POST"} []
[2017-09-12 19:38:47] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:38:47] php.CRITICAL: Fatal Error: Call to a member function loadUserByUsername() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:38:47] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function loadUserByUsername() on null" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 161 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:38:47] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:38:55] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://localhost:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:38:56] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:38:56] php.CRITICAL: Fatal Error: Call to a member function loadUserByUsername() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:38:56] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function loadUserByUsername() on null" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 161 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:38:56] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:44:23] doctrine.DEBUG: "START TRANSACTION" [] []
[2017-09-12 19:44:23] doctrine.DEBUG: INSERT INTO users (username, username_canonical, email, email_canonical, enabled, salt, password, last_login, confirmation_token, password_requested_at, roles) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) {"1":"root","2":"root","3":"root@root.es","4":"root@root.es","5":true,"6":"0CZd7ZcF9L7mYYP7LFV.a7tz1W [...]","7":"9XF1BjBvpr+v6iABXDdUzpJxp3 [...]","8":null,"9":null,"10":null,"11":[]} []
[2017-09-12 19:44:23] doctrine.DEBUG: "COMMIT" [] []
[2017-09-12 19:44:37] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:44:40] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://localhost:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:44:41] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:44:41] php.CRITICAL: Fatal Error: Call to a member function loadUserByUsername() on null {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:44:41] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalErrorException: "Error: Call to a member function loadUserByUsername() on null" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 161 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalErrorException(code: 0): Error: Call to a member function loadUserByUsername() on null at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:161)"} []
[2017-09-12 19:44:41] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:46:16] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://localhost:8000/oauth/v2/token","method":"GET"} []
[2017-09-12 19:48:48] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:48:58] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://localhost:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:48:58] request.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\ContextErrorException: "Catchable Fatal Error: Argument 5 passed to FOS\OAuthServerBundle\Storage\OAuthStorage::__construct() must implement interface Symfony\Component\Security\Core\User\UserProviderInterface, instance of FOS\UserBundle\Doctrine\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined" at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php line 80 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\ContextErrorException(code: 0): Catchable Fatal Error: Argument 5 passed to FOS\\OAuthServerBundle\\Storage\\OAuthStorage::__construct() must implement interface Symfony\\Component\\Security\\Core\\User\\UserProviderInterface, instance of FOS\\UserBundle\\Doctrine\\UserManager given, called in /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/var/cache/dev/appDevDebugProjectContainer.php on line 1776 and defined at /Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/friendsofsymfony/oauth-server-bundle/Storage/OAuthStorage.php:80)"} []
[2017-09-12 19:48:58] php.DEBUG: Warning: stream_context_get_params(): Invalid stream/context parameter {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Caster/ResourceCaster.php\",\"line\":53,\"trace\":[{\"function\":\"handleError\",\"class\":\"Symfony\\\\Component\\\\Debug\\\\ErrorHandler\",\"type\":\"->\"},{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Component/VarDumper/Cloner/AbstractCloner.php\",\"line\":205,\"function\":\"call_user_func\"},{\"function\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\{closure}\",\"class\":\"Symfony\\\\Component\\\\VarDumper\\\\Cloner\\\\AbstractCloner\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:50:46] php.DEBUG: Warning: fsockopen(): unable to connect to 127.0.0.1:8000 (Connection refused) {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\SilencedErrorContext: {\"severity\":2,\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":120,\"trace\":[{\"file\":\"/Applications/XAMPP/xamppfiles/symphonyProjects/api_project/vendor/symfony/symfony/src/Symfony/Bundle/WebServerBundle/WebServerConfig.php\",\"line\":54,\"function\":\"findBestPort\",\"class\":\"Symfony\\\\Bundle\\\\WebServerBundle\\\\WebServerConfig\",\"type\":\"->\"}],\"count\":1})"} []
[2017-09-12 19:50:50] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://localhost:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:50:51] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:50:51] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM users t0 WHERE t0.username_canonical = ? LIMIT 1 ["root"] []
[2017-09-12 19:51:01] request.INFO: Matched route "fos_oauth_server_token". {"route":"fos_oauth_server_token","route_parameters":{"_controller":"fos_oauth_server.controller.token:tokenAction","_route":"fos_oauth_server_token"},"request_uri":"http://localhost:8000/oauth/v2/token","method":"POST"} []
[2017-09-12 19:51:01] doctrine.DEBUG: SELECT t0.random_id AS random_id_1, t0.redirect_uris AS redirect_uris_2, t0.secret AS secret_3, t0.allowed_grant_types AS allowed_grant_types_4, t0.id AS id_5 FROM oauth2_clients t0 WHERE t0.id = ? AND t0.random_id = ? LIMIT 1 ["1","3bcbxd9e24g0gk4swg0kwgcwg4 [...]"] []
[2017-09-12 19:51:01] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM users t0 WHERE t0.username_canonical = ? LIMIT 1 ["admin"] []
[2017-09-12 19:51:01] doctrine.DEBUG: "START TRANSACTION" [] []
[2017-09-12 19:51:01] doctrine.DEBUG: INSERT INTO oauth2_access_tokens (token, expires_at, scope, client_id, user_id) VALUES (?, ?, ?, ?, ?) {"1":"ZTExM2Y5YThhMmU0NDdhYzEzZm [...]","2":1505242261,"3":null,"4":1,"5":1} []
[2017-09-12 19:51:01] doctrine.DEBUG: "COMMIT" [] []
[2017-09-12 19:51:01] doctrine.DEBUG: "START TRANSACTION" [] []
[2017-09-12 19:51:01] doctrine.DEBUG: INSERT INTO oauth2_refresh_tokens (token, expires_at, scope, client_id, user_id) VALUES (?, ?, ?, ?, ?) {"1":"OTgwOWUwYTlmNTA1MTI0NDNkM2 [...]","2":1506448261,"3":null,"4":1,"5":1} []
[2017-09-12 19:51:01] doctrine.DEBUG: "COMMIT" [] []
[2017-09-12 19:51:51] request.INFO: Matched route "app_api_index". {"route":"app_api_index","route_parameters":{"_controller":"AppBundle\\Controller\\ApiController::indexAction","_route":"app_api_index"},"request_uri":"http://localhost:8000/api","method":"GET"} []
[2017-09-12 19:51:51] doctrine.DEBUG: SELECT t0.token AS token_1, t0.expires_at AS expires_at_2, t0.scope AS scope_3, t0.id AS id_4, t0.client_id AS client_id_5, t0.user_id AS user_id_6 FROM oauth2_access_tokens t0 WHERE t0.token = ? LIMIT 1 ["ZTExM2Y5YThhMmU0NDdhYzEzZm [...]"] []
[2017-09-12 19:51:51] doctrine.DEBUG: SELECT t0.username AS username_1, t0.username_canonical AS username_canonical_2, t0.email AS email_3, t0.email_canonical AS email_canonical_4, t0.enabled AS enabled_5, t0.salt AS salt_6, t0.password AS password_7, t0.last_login AS last_login_8, t0.confirmation_token AS confirmation_token_9, t0.password_requested_at AS password_requested_at_10, t0.roles AS roles_11, t0.id AS id_12 FROM users t0 WHERE t0.id = ? [1] []
